admin/client/src/components/PopoverTooltip/PopoverTooltip.js

```diff
@@ -0,0 +1,46 @@
+import $ from 'jQuery';
+import 'tetherWrapper';
+import 'bootstrap-popover';
+import 'bootstrap-tooltip';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class PopoverTooltip extends React.Component {
+  componentDidMount() {
+    const el = ReactDOM.findDOMNode(this);
+
+    // Enable BS Popovers and tooltips
+    $(el).find('[data-toggle="tooltip"]').tooltip();
+    $(el).find('[data-toggle="popover"]').popover({ html: true });
+
+    // Accessibility support for popover focus (requires anchor element)
+    $(el).find('.popover-dismiss').popover({ trigger: 'focus' });
+  }
+
+  // TODO allow for custom popover/tooltip trigger
+  // eg. (classes, contents, data-toggle, placement, icon and text)
+  // Add the possibility to use FormAction component as a trigger
```

Comment:
Rather than creating these temporary components with lots to TODOs, I think we should start using react-bootstrap which already has those TODOs resolved ;) https://react-bootstrap.github.io/components.html#popovers
Maybe that's out of scope for the current sprint, but there's not much point going beyond styling then